cmd/instance/instance_vnc.go

```diff
@@ -75,27 +78,27 @@ Duration follows Go's duration format (e.g. "30m", "1h", "24h")`,
 
 		err = waitEndpointReady(vnc.URI)
 		if err != nil {
-			utility.Error("VNC Console URL is not reachable: %s", err)
+			utility.Error("Console URL is not reachable: %s", err)
 			os.Exit(1)
 		}
 
-		utility.Info("Opening VNC in your default browser...")
+		utility.Info("Opening the Console in your default browser...")
 		time.Sleep(3 * time.Second)
 
 		// Open VNC in the browser
 		err = browser.OpenInBrowser(vnc.URI)
 		if err != nil {
-			utility.Error("Failed to open VNC URL in the browser: %s", err)
+			utility.Error("Failed to open the Console Access URL in the browser: %s", err)
```

Comment:
Would change to "Failed to open the console access URL in the browser:"

Reply:
I've removed the "the"... let me know

Reply:
I would keep it. You're talking about a particular, definite URL.